src/common/low_precision_transformations/src/reduce_sum.cpp

```diff
@@ -69,10 +69,23 @@ void ReduceSumTransformation::changeDequantizationValues(
 
         // (a1 - s) + (a2 - s) + ... + (an - s) = (a1 + a2 + ... + an) - n * s
         const auto reductionSizeConstant = ov::opset1::Constant::create(deqPrecision, Shape{}, { static_cast<float>(reductionSize) });
-        const auto result = fold<ov::opset1::Multiply>(dequantization.subtractConstant, reductionSizeConstant);
+        assert(deqPrecision == dequantization.subtract->get_input_element_type(0));
+        const auto result = fold<ov::opset1::Multiply>(
+            dequantization.subtractConstant->get_element_type() == deqPrecision ?
+                dequantization.subtractConstant :
+                std::dynamic_pointer_cast<ov::Node>(foldConvert(dequantization.subtractConstant, deqPrecision)),
```

Comment:
`foldConvert` skips conversion if requested element type is the same as current one. So we can write shorter
```suggestion
            foldConvert(dequantization.subtractConstant, deqPrecision),
```

Reply:
updated, thanks